cassandra/client.go

```diff
@@ -322,21 +356,17 @@ func addSslOptions(cluster *gocql.ClusterConfig, options *sslOptions) *gocql.Clu
 	return cluster
 }
 
-func initializeSession(cluster *gocql.ClusterConfig) *gocql.Session {
+func initializeSession(cluster *gocql.ClusterConfig, co clientOptions) *gocql.Session {
 	session, err := cluster.CreateSession()
 	if err != nil {
 		log.Fatal(err.Error())
 	}
 
-	if err := session.Query(createKeyspaceCQL).Exec(); err != nil {
-		log.Fatal(err.Error())
-	}
-
-	if err := session.Query(createTableCQL).Exec(); err != nil {
+	if err := session.Query(fmt.Sprintf(createTableCQL, co.keyspace)).Exec(); err != nil {
 		log.Fatal(err.Error())
 	}
 
-	if err := session.Query(createTagTableCQL).Exec(); err != nil {
+	if err := session.Query(fmt.Sprintf(createTagTableCQL, co.keyspace)).Exec(); err != nil {
```

Comment:
@squall0gd, tag table is a by-product. Let us leave it. Also metric as the table name seems fine if we don't want to change too much now.